app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -1068,4 +1050,17 @@ def build_template_filter
 
     ->(_) { true } # do not apply a filter
   end
+
+  def options_for_provisioning(db, report_scopes, report_name)
+    {
+      :model         => db,
+      :gtl_type      => "table",
+      :named_scope   => report_scopes,
+      :report_name   => report_name,
```

Comment:
`report_name` is a constant in both callers .. should this just be a part of the function?

Reply:
Yep, I guess it can be...if needed in future, it can be parameterize again